packages/sdk/src/sudoswap/exchange.ts

```diff
@@ -0,0 +1,56 @@
+import { Signer } from "@ethersproject/abstract-signer";
+import { Contract, ContractTransaction } from "@ethersproject/contracts";
+
+import * as Addresses from "./addresses";
+import { TxData } from "../utils";
+
+import PairFactoryAbi from "./abis/FactoryPair.json";
+
+export class Exchange {
+  public chainId: number;
+  public contract: Contract;
+
+  constructor(chainId: number) {
+    this.chainId = chainId;
+    this.contract = new Contract(
+      Addresses.PairFactory[this.chainId],
+      // @ts-ignore
+      PairFactoryAbi
+    );
+  }
+
+    // --- Deposit NFTs ---
+
+    public async depositNFTs(
+        maker: Signer,
+        nft: string, //contract
+        ids: number[], //tokenId
+        recipient: string //pool
```

Comment:
Feels cleaner to rename this to `pool`.

Reply:
yeah, i agree. i'll change it to `pool`, I had it called `recipient` before 'cause I was using the same variable names that sudoswap used